x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/steps_modal.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+
+import {
+  EuiText,
+  EuiSteps,
+  EuiSpacer,
+  EuiButton,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiTitle,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+
+export interface StepsModalContent {
+  domainId: string;
+  steps: string[];
+  documentationUrl?: string;
+}
+
+interface Props {
+  closeModal: () => void;
+  modalContent: StepsModalContent;
+}
+
+const i18nTexts = {
+  getModalTitle: (domainId: string) =>
+    i18n.translate('xpack.upgradeAssistant.kibanaDeprecations.stepsModal.modalTitle', {
+      defaultMessage: "Fix '{domainId}'",
```

Comment:
Can this be more specific? Again, you're not fixing the plugin, you're fixing a specific instance of a deprecated feature being used.

Reply:
Not currently. For now, I could follow the same suggestion in https://github.com/elastic/kibana/pull/97159#discussion_r626959789.